.idea/terminal.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="TerminalProjectOptionsProvider">
+    <envs>
+      <env key="D:\Program Files\Android\Sdk\platform-t" value="ols=" />
```

Comment:
What is this?... Seem dangerous lol

Reply:
It's IDE's config and can be ignored. I'll delete it and add to .gitignore then.